### PR TITLE
Add more Dashboard docs

### PR DESCRIPTION
**Pull request by breeswish**
Signed-off-by: Breezewish <me@breeswish.org>

### What is changed, added or deleted? (Required)

Added the following new documentations for TiDB Dashboard, to make content more complete:

- TiDB Dashboard Introduction
- Deploy TiDB Dashboard
- Use TiDB Dashboard behind a Reverse Proxy
- Enhance TiDB Dashboard Security
- TiDB Dashboard FAQ

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by lilin90** [comment]
I'll review this PR on Jun 3.

File: dashboard/dashboard-diagnostics-report.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 集群诊断页面
+title: TiDB Dashboard 诊断报告
```
Comment (lilin90) [inline dashboard/dashboard-diagnostics-report.md:2]: Please keep the first-level heading consistent with this `title`.

File: dashboard/dashboard-ops-reverse-proxy.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+   > ```yaml
+   > monitored:
+   >   ...
+   > server_configs:
+   >   tidb: ...
+   >   tikv: ...
+   >   pd:
+   >     dashboard.public-path-prefix: /foo
+   >   ...
+   > ```
```
Comment (lilin90) [inline dashboard/dashboard-ops-reverse-proxy.md:120]: The `>` character is unnecessary? Suggest removing them.

**Comment by lilin90** [comment]
@breeswish I added a commit https://github.com/pingcap/docs-cn/pull/3440/commits/89607b4af66b389fc8f71b4de92f3c255b4eb661 to update wording and format. FYI.

File: dashboard/dashboard-ops-reverse-proxy.md [hunk 3]
```diff
@@ -55,17 +55,17 @@ http://192.168.0.123:2379/dashboard/
```
Comment (breeswish) [inline dashboard/dashboard-ops-reverse-proxy.md:55]: 第 54 行这种步骤 1 是否也需要修改为第 1 步？ @lilin90

**Comment by breeswish** [comment]
@HunDunDM PTAL

File: dashboard/dashboard-ops-reverse-proxy.md (hunk 3 above)
Comment (lilin90) [reply dashboard/dashboard-ops-reverse-proxy.md:55]: @breeswish 不需要的，只在 heading 里用“第 1 步”即可，正文具体步骤用 1, 2, 3, ... 便于官网解析呈现。

File: dashboard/dashboard-faq.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+3. 对于现有集群，可通过再次启动集群的方法汇报监控地址（将 `CLUSTER_NAME` 替换为实际集群名称）：
+
+   ```bash
+   tiup cluster start CLUSTER_NAME
+   ```
+
+   即使集群已经启动，请仍然执行该命令。该命令不会影响集群上正常的业务，但会刷新并上报监控地址，从而能让监控在 TiDB Dashbaord 中正常显示。
+
+### 概况页面中 Top SQL、最近慢查询显示 `invalid connection` 错误
```
Comment (HunDunDM) [inline dashboard/dashboard-faq.md:53]:
```suggestion
### 概况页面中 Top SQL 语句、最近慢查询显示 `invalid connection` 错误
```

File: dashboard/dashboard-faq.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+可能的原因是你开启了 TiDB 的 `prepared-plan-cache` 功能。在某些版本的 TiDB 中，`prepared-plan-cache` 作为实验性功能存在一些缺陷，开启后可能会导致 TiDB Dashboard（及其他应用）出现该问题。请依据[文档](/tidb-configuration-file.md#prepared-plan-cache)关闭 `prepared-plan-cache` 功能。
```
Comment (HunDunDM) [inline dashboard/dashboard-faq.md:55]:
```suggestion
可能的原因是你开启了 TiDB 的 `prepared-plan-cache` 功能。作为实验性功能，`prepared-plan-cache` 在某些版本的 TiDB 中存在一些缺陷，开启后可能会导致 TiDB Dashboard（及其他应用）出现该问题。请依据[文档](/tidb-configuration-file.md#prepared-plan-cache)关闭 `prepared-plan-cache` 功能。
```

File: dashboard/dashboard-ops-reverse-proxy.md (hunk 3 above)
Comment (breeswish) [reply dashboard/dashboard-ops-reverse-proxy.md:55]: 这里第 54 行指代的即是 heading 第 1 步那部分里的内容，是不是改掉好一些，怕读者不知道它指代什么

File: dashboard/dashboard-ops-reverse-proxy.md (hunk 2 above)
Comment (breeswish) [reply dashboard/dashboard-ops-reverse-proxy.md:120]: 它们主要是用作补充说明，便于用户校对修改是否正确，应该问题不大

File: dashboard/dashboard-statement-details.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
 ---
-title: 查看执行详情
+title: TiDB Dashboard SQL 语句分析执行详情页面
 summary: 查看单个 SQL 语句执行的详细情况
 category: how-to
```
Comment (TomShawn) [inline dashboard/dashboard-statement-details.md:4]:
```suggestion
category: how-to
aliases: ['/docs-cn/dev/dashboard/dashboard-statement-detail/']
```

File: dashboard/dashboard-ops-reverse-proxy.md (hunk 2 above)
Comment (lilin90) [reply dashboard/dashboard-ops-reverse-proxy.md:120]: @breeswish  通常建议格式尽量简单，以确保官网解析正常。可以观察下，如果在官网格式错乱了，就请删掉这些字符。 @TomShawn 英文版里不必留这些字符。

Comment (TomShawn) [reply dashboard/dashboard-ops-reverse-proxy.md:120]: 好的

**Pull request merged**